homeassistant/components/plugwise/sensor.py

```diff
@@ -31,27 +32,49 @@
         native_unit_of_measurement=TEMP_CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+    ),
+    SensorEntityDescription(
+        key="setpoint_high",
+        name="Cooling setpoint",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
```

Comment:
- Why is this one disabled?
- No state class?

Reply:
As above for setpoint_high.

Reply:
Yes, but it is marked diagnostic already, so there should be no need to disable it.

Reply:
I prefer to remove the entity-category.

Reply:
That would  be false, as this is not a primary entity.